src/components/Sidebar/NavContent.tsx

```diff
@@ -70,90 +31,117 @@ const NavContent: React.FC<NavContentProps> = ({ handleDrawerToggle }) => {
+
                   <NavItem
-                    href="https://synapseprotocol.com/?inputCurrency=gOHM&outputCurrency=gOHM&outputChain=43114"
                     icon="bridge"
                     label={t`Bridge`}
+                    href="https://synapseprotocol.com/?inputCurrency=gOHM&outputCurrency=gOHM&outputChain=43114"
                   />
                 </>
               )}
-              {}
-              {Object.keys(externalUrls).map((link: any, i: number) => (
-                <NavItem
-                  key={i}
-                  href={`${externalUrls[link].url}`}
-                  icon={externalUrls[link].icon as any}
-                  label={externalUrls[link].title as any}
-                />
-              ))}
+
+              <NavItem href="https://forum.olympusdao.finance/" icon="forum" label={t`Forum`} />
+
+              <NavItem href="https://vote.olympusdao.finance/" icon="governance" label={t`Governance`} />
+
+              <NavItem href="https://docs.olympusdao.finance/" icon="docs" label={t`Docs`} />
+
+              <NavItem href="https://immunefi.com/bounty/olympus/" icon="bug-report" label={t`Bug Bounty`} />
+
+              <NavItem href="https://grants.olympusdao.finance/" icon="grants" label={t`Grants`} />
             </div>
           </div>
         </div>
-        <Box className="dapp-menu-social" display="flex" justifyContent="space-between" flexDirection="column">
-          <Social />
+
+        <Box display="flex" justifyContent="space-between" paddingX="50px" paddingY="24px">
+          <Link href="https://github.com/OlympusDAO" target="_blank">
+            <Icon name="github" className={classes.gray} />
+          </Link>
+
+          <Link href="https://olympusdao.medium.com/" target="_blank">
+            <Icon name="medium" className={classes.gray} />
+          </Link>
+
+          <Link href="https://twitter.com/OlympusDAO" target="_blank">
+            <Icon name="twitter" className={classes.gray} />
+          </Link>
+
+          <Link href="https://discord.gg/6QjjtUcfM4" target="_blank">
+            <Icon name="discord" className={classes.gray} />
+          </Link>
         </Box>
       </Box>
     </Paper>
   );
 };
 
+const Bonds: React.VFC<{ isInverseBond?: boolean }> = ({ isInverseBond = false }) => {
+  const bonds = useBonds({ isInverseBond }).data;
+
+  if (!bonds) return null;
+
+  return (
+    <>
+      {sortByDiscount(bonds)
+        .filter(bond => !bond.isSoldOut)
+        .map(bond => (
+          <NavItem
+            key={bond.id}
+            label={bond.quoteToken.name}
+            to={isInverseBond ? `/bonds/inverse/${bond.id}` : `/bonds/${bond.id}`}
+            chipColor={bond.discount.gt(new DecimalBigNumber("0")) ? "success" : "error"}
+            chip={`${bond.discount.mul(new DecimalBigNumber("100")).toString({ decimals: 2, trim: true })}%`}
+          />
+        ))}
+    </>
+  );
+};
+
 export default NavContent;
```

Comment:
@sam-potter  - sorry to mess ya up here, but there were some back and forths on the nav updates. Could you take a look at the latest changes in develop. We'll have to resolve conflicts/refactor a little bit here since we're moving back to the previous DisplayBondDiscount implementation

Reply:
all good, resolved it now